app/assets/stylesheets/components/_header.scss

```diff
@@ -1,21 +1,173 @@
+// Blue box icons in the header, extend as needed. If used elsewhere, abstract from here
+.header-icons {
+  @extend %border-radius;
+  @extend .btn--primary;
+  right: 0;
+  height: 40px;
+  padding: 0;
+  position: relative;
+  float: right;
+  margin-left: $baseline-unit*2;
+  color: #fff;
+  @at-root button.header-icons, & { /* overriding button.btn in _buttons.scss */
+    width: 40px;
+  }
+  .svg-icon {
+    width: 60%;
+  }
+}
+
 .header {
   position: relative;
-  background: #fff;
   z-index: 15;
+  height: 87px;
+  .l-constrained {
+    position: fixed;
+    background: #fff;
+  }
+  @include respond-to($mq-m) {
+    height: auto;
+    .l-constrained {
+      position: relative;
+    }
+  }
 }
 
-.mobile-nav-button {
-  display: block;
-
+.header-content {
+  @extend %clearfix; 
+  box-sizing: border-box;
+  display: flex;
```

Comment:
May need to consider fallback for browsers that don't use flexbox.

Reply:
@davidtrussler As discussed with yourself and @johnplayer1982 we'll leave this change